files/default/neutron-ha-tool.py

```diff
@@ -404,9 +379,31 @@ def replicate_dhcp(qclient, noop=False):
     LOG.info("Added %s networks to DHCP agents", added)
 
 
+def migrate_l3_routers_from_agent(qclient, agent_id, target_ids, noop):
+    LOG.info("Querying agent_id=%s for routers to migrate away", agent_id)
+    router_id_list = list_routers_on_l3_agent(qclient, agent_id)
+
+    if not noop:
```

Comment:
Sure, but I still can't see any problems.  If you're convinced about this, please point out a specific log message which used to appear with `--noop` and doesn't any more.